src/playermat/Playmat.ttslua

```diff
@@ -424,32 +409,54 @@ end
 
 function markSpawned(object)
   local saved = DATA_HELPER.call('setSpawnedPlayerCardGuid', {object.getGUID(), true})
-  if not saved then
-    error('attempt to mark player card spawned before data loaded')
-  end
+  if not saved then error('attempt to mark player card spawned before data loaded') end
 end
 
+-- contains position and amount of boxes for the upgradesheets that change uses
+-- Alchemical Distillation, Damning Testimony, Living Ink and Hyperphysical Shotcaster
+local customizationsTable = {
+  ["09040"] = {5, 2},
+  ["09059"] = {2, 2},
+  ["09079"] = {3, 2},
+  ["09119"] = {6, 4}
+}
+
 function spawnTokensFor(object)
   local cardMetadata = JSON.decode(object.getGMNotes()) or {}
-  local token = nil
   local type = nil
+  local token = nil
   local tokenCount = 0
-  if (cardMetadata.uses ~= nil) then
-    for i, useInfo in ipairs(cardMetadata.uses) do
-      token = useInfo.token
+  if cardMetadata.uses ~= nil then
+    for _, useInfo in ipairs(cardMetadata.uses) do
       type = useInfo.type
+      token = useInfo.token
       tokenCount = useInfo.count
-      if (activeInvestigatorId == "03004" and useInfo.type == "Charge") then
-        tokenCount = tokenCount + 1
+    end
+
+    -- additional uses for certain customizable cards (by checking the upgradesheets)
+    if customizationsTable[cardMetadata.id] ~= nil then
```

Comment:
Let's make sure we remember this is here when we get around to pushing the actual metadata changes

Reply:
As long as the ID in the metadata doesn't get changed, this should not need any changes in the future
![image](https://user-images.githubusercontent.com/97286811/202253431-c935ab59-27e1-4620-9028-9a98736fd5de.png)

Reply:
We'd talked about having the metadata actually changed based on the selected boxes.  If we do that, the modification here would duplicate it.  I think changing the metadata will be a more general solution, otherwise everywhere that uses metadata will have to have these sorts of checks.  But we can change this later when we implement that.